### PR TITLE
docs(guardrails): write how-to

### PR DESCRIPTION
**Pull request by konrad-czarnota-ds**

### CODE REVIEW

File: docs/how-to/use_guardrails.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# How-To: Use Guardrails
+
+ragbits offers an expandable guardrails system. You can use one of the available guardrails or create your own to prevent toxic language, PII leaks etc.
```
Comment (ludwiktrammer) [inline docs/how-to/use_guardrails.md:3]: We decided to always write "db-ally" starting with a lowercase letter but does this also extend to "ragbits"? Looks a little weird to me

File: docs/how-to/use_guardrails.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    manager = GuardrailManager([OpenAIModerationGuardrail()])
+    return await manager.verify(message)
+
+
+if __name__ == '__main__':
+    print(asyncio.run(verify_message("Test message")))
+```
+
+The expected output has the following structure:
+```python
```
[the remark this answers is not in the document]
Comment (konrad-czarnota-ds) [reply docs/how-to/use_guardrails.md:28]: I agree that linking is the best option, but I would like to leave something here before we have API reference. Would adding a line like: `The expected output structure is defined in `GuardrailVerificationResult` class that is defined in `packages/ragbits-guardrails/src/ragbits/base`.` be good for you or should I delete this part completely?

[the remark this answers is not in the document]
Comment (ludwiktrammer) [reply docs/how-to/use_guardrails.md:28]: I wouldn't like to the source. Before we have API docs maybe just write this:

> The expected output is an object with the following properites:
> ```
>   guardrail_name: str
>   succeeded: bool
>   fail_reason: str | None
> ```

What do you think?

File: docs/how-to/use_guardrails.md (hunk 1 above)
Comment (konrad-czarnota-ds) [reply docs/how-to/use_guardrails.md]: Changed to capital letter

**Review by ludwiktrammer — APPROVED** [review]

**Pull request merged**